### ISSUE TITLE
Is it possible to use special characters in attribute when parsing? 

### ISSUE THREAD

**Issue by sirsmac** [technosophos/querypath#157]
I have some custom tags for special purposes. I'm using special characters in attribute but QP does not allow me to use it. See html side example below;

<spml:each var="My_variable" idx="Index" in="{(0..<5)}" hideOnLast="last">
        <div>
            Each: Row {Index}: {My_variable}
        </div>
        <div id="last" class="last">
            {Index}<hr />
        </div>
</spml:each>

I'm using qp() for parsing the html source. I also have tried with htmlqp() as well.  
Can you please fix it? Or is there any workaround for this?

Thanks
Smac

**Comment by technosophos** [comment]
Is it the `in` attribute that's not working? What do you get back? I think it is probably the `<` that is causing problems because the XML and HTML specs both require that to be escaped.

**Comment by sirsmac** [comment]
Yes exactly that's the problem. Is there any solution for allowing these type of special characters in attributes?
